src/styles/main.scss

```diff
@@ -10,5 +10,83 @@ body {
   box-sizing: border-box;
 }
 
-@import "_header.scss";
-@import "_catalog.scss";
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+  justify-content: center;
+  align-content: center;
+
+  &__card {
+    display: grid;
+    justify-content: center;
+    align-content: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (max-width: 487px) {
+    & {
+      grid-template-columns: repeat(1, 200px);
+    }
+  }
```

Comment:
You don't need to use `&` for media in scss.
You can simply write under necessary selector.
Here you apply it to `.catalog`, so change these and put them right after line 18-19.

```
@media (condition) {
  some: styles;
}
```